backend/integrations/discord/views.py

```
@@ -99,8 +104,10 @@ async def check_verified(  # type: ignore[override]
             await send_final_handoff_dm(interaction.user)
             try:
                 await interaction.message.edit(view=self)
+            except discord.NotFound:
+                logger.debug(f"Message not found when editing onboarding view for user {interaction.user.id}")
             except Exception:
-                pass
+                logger.exception(f"Unexpected error editing onboarding view")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)